templates/services/portainer/portainer_yml.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+#create portainer.yml
+
+mkdir -p /srv/portainer
+
+echo "services:"
+  echo "  portainer:"
+  echo "    image: treehouses/portainer"
```

Comment:
Similar to `nextcloud_yml.sh`, the image we want to pull isn't `treehouses/portainer`. It's `portainer/portainer`.

https://hub.docker.com/r/portainer/portainer